uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/resources/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants_de.properties

```diff
@@ -0,0 +1,4 @@
+MetaFileEditor=Meta Datei Editor
```

Comment:
Should this be `Metadatei-Editor` ?

Reply:
This came from previous files, I just moved it, let me know if you think that I should modify it